css/styles.css

```diff
@@ -0,0 +1,165 @@
+* {
+  font-family: Merriweather, Source Sans Pro, Oswald;
+  font-weight: lighter;
+}
+
+header {
+  background-color: black;
+  float: left;
+  width: 100%;
+  border-bottom: 3px solid yellow;
+  color: white;
+}
+
+nav {
+  text-align: right;
+  padding: 10px;
+}
+
+nav a {
+  color: white;
+  list-style: none;
+  padding-right: 25px;
+  text-decoration: none;
+}
+
+article {
+  background-color: red;
+  clear: both;
+}
+
+aside {
+  clear: both;
+  height: 45vh;
```

Comment:
@kvelez1 remove the height, let the height of it's children determine the height 🔨